### PR TITLE
Add close button to server error modal

### PR DESCRIPTION
**Pull request by kennyt**
pic

### CODE REVIEW

**Comment by rcorral** [comment]
@kennyt could you also improve the paddings/margins of the modal in this PR? There's waaay too much spacing around elements.

**Comment by kennyt** [comment]
@rcorral design said the spacing is fine for now, until we do a modal overhaul

File: src/js/components/ServerErrorModal.js [hunk 1]
```diff
@@ -62,18 +63,34 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
...
+          Close
+        </div>
+      </div>
+    );
+  }
+
   getContent() {
-    let errors = this.state.errors.map(function (error, i) {
+    let {errors} = this.state;
+    let errorMessages = this.state.errors.map(function (error, i) {
```
Comment (mlunoe) [inline src/js/components/ServerErrorModal.js]: Let's make use of the variable you created

File: src/js/components/ServerErrorModal.js [hunk 2]
```diff
@@ -62,18 +63,34 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
...
+      let errorMessageClass = classNames('text-align-center', {
+        // Last error message doesn't have margin bottom.
+        'flush-bottom': i === errors.length - 1
```
Comment (mlunoe) [inline src/js/components/ServerErrorModal.js]: Could we create a variable for the length to not re-compute it on every iteration?

**Comment by mlunoe** [comment]
Sorry, didn't notice these few nit-picks until now

**Comment by jfurrow** [comment]
@kennyt bump :)

**Pull request merged**